services/service_now.md

```diff
@@ -0,0 +1,48 @@
+# Service Now tips and tricks
+
+Service Now offers two ways to access information its database: "canned" views and "bare-bones" views. The two views offer different UIs, different options, and different default data sets. You cannot navigate from one view to the other (at least, I have not found a way). Here's a single ticket shown both ways:
```

Comment:
You can get from canned to bare bones via switch to classic view under my account menu.  I do not think you can go back

Reply:
Added in a312fd0bd8d099335d2df95915771141f3b362b6.